### ISSUE TITLE
installed package libelf cannot be found in remote list!

### ISSUE THREAD

**Issue by hrw** [openwrt/asu#604]
23.05 got released so I decided to upgrade my AP to latest 22.03 (from older build) and it went fine. 

But one thing worried me:
```
root@krolik:~# auc
auc/0.3.1-1
Server:    https://sysupgrade.openwrt.org
Running:   22.03.5 r20134-5f15225c1e on mediatek/mt7622 (linksys,e8450-ubi)
Available: 22.03.5 r20134-5f15225c1e
Requesting package lists...
installed package libelf cannot be found in remote list!
installed package libncurses cannot be found in remote list!
Are you sure you want to continue the upgrade process? [N/y] n
```

System has libelf**1** nd libncurses**6** installed:
```
root@krolik:~# opkg list_installed|grep libelf ; opkg list_installed|grep libncurses
libelf1 - 0.186-1
libncurses6 - 6.3-2
```

And it is not even some reminder from past:
```
root@krolik:~# grep ^Package /usr/lib/opkg/status |grep libelf
Package: libelf1
root@krolik:~# grep ^Package /usr/lib/opkg/status |grep libncurses
Package: libncurses6
root@krolik:~#
```

Safe to ignore probably but surprised me.

**Comment by edrikk** [comment]
I would not use that - in fact both auc and asu are broken right now..  So ticket just a couple below yours.

**Comment by aparcar** [comment]
AUC should be fixed, please give it another go

**Comment by hrw** [comment]
Looks good.

```
root@krolik:~# auc -b 23.05
auc/0.3.1-1
Server:    https://sysupgrade.openwrt.org
Running:   22.03.5 r20134-5f15225c1e on mediatek/mt7622 (linksys,e8450-ubi)
Available: 23.05.0 r23497-6637af95aa
Requesting package lists...
 kmod-usb-storage: 5.10.176-1 -> 5.15.134-1
 terminfo: 6.3-2 -> 6.4-2
 iwinfo: 2022-12-15-8d158096-1 -> 2023-07-01-ca79f641-1
 luci-app-statistics: git-23.153.53801-38f5b55 -> git-23.184.27766-2c7b34b
 luci-lib-base: git-20.232.39649-1f6dc29 -> git-22.308.54612-9118452
 kmod-crypto-gf128: 5.10.176-1 -> 5.15.134-1
 opkg: 2022-02-24-d038e5b6-1 -> 2022-02-24-d038e5b6-2
 kmod-usb-core: 5.10.176-1 -> 5.15.134-1
 luci-app-opkg: git-23.093.42303-e16f620 -> git-23.009.82915-ec3aac4
 kmod-usb-storage-uas: 5.10.176-1 -> 5.15.134-1
 ubus: 2022-06-01-2bebf93c-1 -> 2023-06-05-f787c97b-1
 iw: 5.16-1 -> 5.19-1
 kmod-crypto-manager: 5.10.176-1 -> 5.15.134-1
 libuci: 2021-10-22-f84f49f0-6 -> 2023-08-10-5781664d-1
 rpcd: 2022-12-15-7de4820c-1 -> 2023-07-01-c07ab2f9-1
 busybox: 1.35.0-5 -> 1.36.1-1
 kmod-crypto-ctr: 5.10.176-1 -> 5.15.134-1
 ip-full: 5.15.0-4 -> 6.3.0-1
 kmod-nft-fib: 5.10.176-1 -> 5.15.134-1
 kmod-nfnetlink: 5.10.176-1 -> 5.15.134-1
 libubus-lua: 2022-06-01-2bebf93c-1 -> 2023-06-05-f787c97b-1
 kmod-crypto-hash: 5.10.176-1 -> 5.15.134-1
 kmod-nf-reject6: 5.10.176-1 -> 5.15.134-1
 kmod-crypto-aead: 5.10.176-1 -> 5.15.134-1
 libiwinfo-lua: 2022-12-15-8d158096-1 -> 2023-07-01-ca79f641-1
 luci-mod-system: git-23.118.78821-0c02883 -> git-23.118.78765-58f7b27
 openssl-util: 1.1.1w-1 -> 3.0.11-1
 kmod-nf-flow: 5.10.176-1 -> 5.15.134-1
 libnl-tiny: 2021-11-21-8e0555fb-1 -> 2023-07-27-bc92a280-1
 kmod-lib-crc-ccitt: 5.10.176-1 -> 5.15.134-1
 getrandom: 2021-08-03-205defb5-2 -> 2022-08-13-4c7b720b-2
 ucode-mod-ubus: 2022-12-02-46d93c9c-1 -> 2023-06-06-c7d84aae-1
 blockdev: 2.37.4-1 -> 2.39-2
 libustream-openssl: 2022-12-08-9217ab46-2 -> 2023-02-25-498f6e26-1
 luci-theme-bootstrap: git-23.093.42704-b47268a -> git-23.085.34270-d94a728
 kmod-pppoe: 5.10.176-1 -> 5.15.134-1
 libbpf: 2022-03-08-04c465fd-1 -> 1.2.2-1
 kmod-pppox: 5.10.176-1 -> 5.15.134-1
 kmod-nf-reject: 5.10.176-1 -> 5.15.134-1
 procd-ujail: 2022-06-01-7a009685-2 -> 2023-06-25-2db83655-2
 base-files: 1502-r20134-5f15225c1e -> 1545-r23497-6637af95aa
 kmod-lib-crc16: 5.10.176-1 -> 5.15.134-1
 kmod-nf-nat: 5.10.176-1 -> 5.15.134-1
 kmod-crypto-crc32c: 5.10.176-1 -> 5.15.134-1
 ucode-mod-uci: 2022-12-02-46d93c9c-1 -> 2023-06-06-c7d84aae-1
 kmod-mt7615e: 5.10.176+2022-09-06-d7054646-5 -> 5.15.134+2023-08-14-b14c2351-1
 netifd: 2022-08-25-76d2d41b-1 -> 2023-09-19-7a58b995-1
 libubus: 2022-06-01-2bebf93c-1 -> 2023-06-05-f787c97b-1
 firewall4: 2022-10-18-7ae5e14b-1 -> 2023-03-23-04a06bd7-1
 uboot-envtools: 2022.01-38 -> 2023.04-1
 kmod-usb-xhci-hcd: 5.10.176-1 -> 5.15.134-1
 dnsmasq: 2.86-16 -> 2.89-4
 procd: 2022-06-01-7a009685-2 -> 2023-06-25-2db83655-2
 kmod-crypto-hmac: 5.10.176-1 -> 5.15.134-1
 kmod-crypto-sha256: 5.10.176-1 -> 5.15.134-1
 kmod-mt7615-common: 5.10.176+2022-09-06-d7054646-5 -> 5.15.134+2023-08-14-b14c2351-1
 ubusd: 2022-06-01-2bebf93c-1 -> 2023-06-05-f787c97b-1
 hostapd-utils: 2022-01-16-cff80b4f-16.2 -> 2023-09-08-e5ccbfc6-4
 kmod-nf-nat6: 5.10.176-1 -> 5.15.134-1
 kmod-lib-crc32c: 5.10.176-1 -> 5.15.134-1
 libelf: 0.186-1 -> 0.189-1
 luci-mod-status: git-23.158.78816-1c26abc -> git-23.236.53405-9b3c7d3
 libjson-script: 2022-05-15-d2223ef9-1 -> 2023-05-23-75a3b870-1
 kmod-mt76-connac: 5.10.176+2022-09-06-d7054646-5 -> 5.15.134+2023-08-14-b14c2351-1
 block-mount: 2022-06-02-93369be0-2 -> 2023-02-28-bfe882d5-1
 kmod-crypto-seqiv: 5.10.176-1 -> 5.15.134-1
 kmod-fs-vfat: 5.10.176-1 -> 5.15.134-1
 kmod-nft-nat: 5.10.176-1 -> 5.15.134-1
 kmod-usb3: 5.10.176-1 -> 5.15.134-1
 collectd-mod-interface: 5.12.0-34 -> 5.12.0-49
 luci-ssl-openssl: git-17.031.53232-b6341bd -> git-23.024.33244-34dee82
 luci-app-firewall: git-23.093.42704-1c01c64 -> git-23.208.40260-9504081
 kmod-thermal: 5.10.176-1 -> 5.15.134-1
 kmod-nf-ipt: 5.10.176-1 -> 5.15.134-1
 collectd-mod-network: 5.12.0-34 -> 5.12.0-49
 jansson: 2.13.1-2 -> 2.14-3
 tcpdump: 4.9.3-4 -> 4.99.4-1
 wpad-openssl: 2022-01-16-cff80b4f-16.2 -> 2023-09-08-e5ccbfc6-4
 ubi-utils: 2.1.4-1 -> 2.1.5-1
 odhcp6c: 2022-08-05-7d21e8d8-18 -> 2023-05-12-bcd28363-20
 fstools: 2022-06-02-93369be0-2 -> 2023-02-28-bfe882d5-1
 kmod-crypto-ghash: 5.10.176-1 -> 5.15.134-1
 uci: 2021-10-22-f84f49f0-6 -> 2023-08-10-5781664d-1
 kmod-fs-ext4: 5.10.176-1 -> 5.15.134-1
 ucode-mod-fs: 2022-12-02-46d93c9c-1 -> 2023-06-06-c7d84aae-1
 collectd: 5.12.0-34 -> 5.12.0-49
 dropbear: 2022.82-2 -> 2022.82-5
 kmod-hwmon-core: 5.10.176-1 -> 5.15.134-1
 kmod-nls-utf8: 5.10.176-1 -> 5.15.134-1
 luci-theme-openwrt-2020: git-23.093.42303-f4d5a1a -> git-23.038.34919-1597d8e
 libnftnl: 1.2.1-2 -> 1.2.6-1
 kmod-nls-cp437: 5.10.176-1 -> 5.15.134-1
 rpcd-mod-file: 2022-12-15-7de4820c-1 -> 2023-07-01-c07ab2f9-1
 odhcpd-ipv6only: 2023-01-02-4a673e1c-2 -> 2023-06-24-52112643-1
 procd-seccomp: 2022-06-01-7a009685-2 -> 2023-06-25-2db83655-2
 libiwinfo-data: 2022-12-15-8d158096-1 -> 2023-07-01-ca79f641-1
 ucode: 2022-12-02-46d93c9c-1 -> 2023-06-06-c7d84aae-1
 rpcd-mod-luci: 20210614 -> 20230123-1
 kmod-mt7915e: 5.10.176+2022-09-06-d7054646-5 -> 5.15.134+2023-08-14-b14c2351-1
 kmod-crypto-ccm: 5.10.176-1 -> 5.15.134-1
 kmod-nf-log: 5.10.176-1 -> 5.15.134-1
 blockd: 2022-06-02-93369be0-2 -> 2023-02-28-bfe882d5-1
 ppp: 2.4.9.git-2021-01-04-3 -> 2.4.9.git-2021-01-04-4
 luci-base: git-23.119.80898-65ef406 -> git-23.236.53405-fc638c8
 collectd-mod-uptime: 5.12.0-34 -> 5.12.0-49
 kmod-leds-gpio: 5.10.176-1 -> 5.15.134-1
 kmod-gpio-button-hotplug: 5.10.176-3 -> 5.15.134-3
 logd: 2021-08-03-205defb5-2 -> 2022-08-13-4c7b720b-2
 kmod-nf-log6: 5.10.176-1 -> 5.15.134-1
 kmod-mac80211: 5.10.176+5.15.92-1-1 -> 5.15.134+6.1.24-3
 libopenssl: 1.1.1w-1 -> 3.0.11-1
 collectd-mod-iwinfo: 5.12.0-34 -> 5.12.0-49
 libblobmsg-json: 2022-05-15-d2223ef9-1 -> 2023-05-23-75a3b870-1
 openwrt-keyring: 2022-03-25-62471e69-3 -> 2022-03-25-62471e69-2
 luci-app-attendedsysupgrade: git-23.093.42303-58b861d -> git-23.132.65998-fa9fb2f
 jshn: 2022-05-15-d2223ef9-1 -> 2023-05-23-75a3b870-1
 nftables-json: 1.0.2-2.1 -> 1.0.8-1
 kmod-ipt-core: 5.10.176-1 -> 5.15.134-1
 attendedsysupgrade-common: 7 -> 8
 kmod-ppp: 5.10.176-1 -> 5.15.134-1
 kmod-nft-offload: 5.10.176-1 -> 5.15.134-1
 libopenssl-conf: 1.1.1w-1 -> 3.0.11-1
 kmod-fs-f2fs: 5.10.176-1 -> 5.15.134-1
 libncurses: 6.3-2 -> 6.4-2
 kmod-crypto-null: 5.10.176-1 -> 5.15.134-1
 collectd-mod-load: 5.12.0-34 -> 5.12.0-49
 libpcap: 1.10.1-4 -> 1.10.4-1
 libiwinfo: 2022-12-15-8d158096-1 -> 2023-07-01-ca79f641-1
 libjson-c: 0.15-2 -> 0.16-3
 kmod-mt7615-firmware: 5.10.176+2022-09-06-d7054646-5 -> 5.15.134+2023-08-14-b14c2351-1
 auc: 0.3.1-1 -> 0.3.2-1
 kmod-nf-conntrack: 5.10.176-1 -> 5.15.134-1
 zlib: 1.2.11-6 -> 1.2.13-1
 kmod-crypto-gcm: 5.10.176-1 -> 5.15.134-1
 kmod-fs-autofs4: 5.10.176-1 -> 5.15.134-1
 luci-lib-nixio: git-20.234.06894-c4a4e43 -> git-22.222.71555-88b9088
 libubox: 2022-05-15-d2223ef9-1 -> 2023-05-23-75a3b870-1
 rpcd-mod-rpcsys: 2022-12-15-7de4820c-1 -> 2023-07-01-c07ab2f9-1
 collectd-mod-rrdtool: 5.12.0-34 -> 5.12.0-49
 kmod-nls-iso8859-1: 5.10.176-1 -> 5.15.134-1
 kmod-crypto-cmac: 5.10.176-1 -> 5.15.134-1
 luci-lib-jsonc: git-22.097.61921-7513345 -> git-22.079.54693-45f411b
 collectd-mod-memory: 5.12.0-34 -> 5.12.0-49
 kmod-crypto-rng: 5.10.176-1 -> 5.15.134-1
 luci: git-23.093.42303-d58cd69 -> git-23.051.66410-a505bb1
 kmod-nf-conntrack6: 5.10.176-1 -> 5.15.134-1
 libgcc: 11.2.0-4 -> 12.3.0-4
 libucode: 2022-12-02-46d93c9c-1 -> 2023-06-06-c7d84aae-1
 kmod-mt76-core: 5.10.176+2022-09-06-d7054646-5 -> 5.15.134+2023-08-14-b14c2351-1
 ubox: 2021-08-03-205defb5-2 -> 2022-08-13-4c7b720b-2
 kmod-crypto-crc32: 5.10.176-1 -> 5.15.134-1
 rpcd-mod-iwinfo: 2022-12-15-7de4820c-1 -> 2023-07-01-c07ab2f9-1
 luci-mod-network: git-23.170.81153-99d3815 -> git-23.233.52805-3a0ef93
 kmod-nft-core: 5.10.176-1 -> 5.15.134-1
 libltdl: 2.4.6-2 -> 2.4.7-1
 kmod-nls-base: 5.10.176-1 -> 5.15.134-1
 mtr-json: 0.95-2 -> 0.95-3
 hostapd-common: 2022-01-16-cff80b4f-16.2 -> 2023-09-08-e5ccbfc6-4
 iperf3: 3.11-1 -> 3.15-1
 collectd-mod-cpu: 5.12.0-34 -> 5.12.0-49
 kmod-ipt-offload: 5.10.176-1 -> 5.15.134-1
 wireless-regdb: 2023.05.03-1 -> 2023.09.01-1
 kmod-scsi-core: 5.10.176-1 -> 5.15.134-1
 kmod-slhc: 5.10.176-1 -> 5.15.134-1
 kmod-cfg80211: 5.10.176+5.15.92-1-1 -> 5.15.134+6.1.24-3
 ppp-mod-pppoe: 2.4.9.git-2021-01-04-3 -> 2.4.9.git-2021-01-04-4
Are you sure you want to continue the upgrade process? [N/y] n
```
